common/lib/local_xblock/xblock_discussion/setup.py

```diff
@@ -0,0 +1,31 @@
+"""Setup for discussion-forum XBlock."""
```

Comment:
@pomegranited In two words, there was a clash with xblock module from XBlock, and to solve it I renamed folder to local_xblock; which actually didn't help, I've just renamed it back and it works.